suppressible-error-prone/src/test/java/com/palantir/suppressibleerrorprone/RemoveRolloutSuppressionsTest.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.suppressibleerrorprone;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import org.junit.jupiter.api.Test;
+
+class RemoveRolloutSuppressionsTest {
+
+    @Test
+    void testRemoveAllForRollout() {
```

Comment:
`test` prefixes on unit tested code is not necessary and just add visual noise, we also prefer to use snake_case for test names and write them as if they were an english sentence (eg `no_specific_checks_means_rollout_suppressions_are_removed` or something similar (tbh `remove_all_for_rollout` would probably also do)).

Reply:
Oh good point - will update 👍